tests/docker_test/docker_test.py

```diff
@@ -16,20 +16,6 @@
 )
 
 
-@pytest.fixture()
-def app():
-    from airflow.www.app import create_app
-
-    app = create_app(testing=True)
-    yield app
-
-
-@pytest.fixture(autouse=True)
-def app_context(app):
-    with app.app_context():
-        yield
-
-
```

Comment:
dead code removal?

Reply:
It had been added only for making the Flask request context work in tests in #131 . So yes, it doesn't serve any purpose anymore.